File: sandbox/python/src/AccountVerifyDefaultExample.py
from datetime import date, datetime
from pprint import pprint

from dropbox_sign import ApiClient, ApiException, Configuration, api, models

configuration = Configuration()

with ApiClient(configuration) as api_client:
    account_verify_request = models.AccountVerifyRequest(
        email_address="some_user@dropboxsign.com",
    )

    try:
        response = api.AccountApi(api_client).account_verify(
            account_verify_request=account_verify_request,
        )

        pprint(response)
    except ApiException as e:
        print("Exception when calling Account#account_verify: %s\n" % e)

File: sandbox/dotnet/src/Dropbox.SignSandbox/AccountVerifyDefaultExample.cs
using System;
using System.Collections.Generic;
using System.IO;

using Dropbox.Sign.Api;
using Dropbox.Sign.Client;
using Dropbox.Sign.Model;

namespace Dropbox.SignSandbox;

public class AccountVerifyDefaultExample
{
    public static void Run()
    {
        var config = new Configuration();

        var accountVerifyRequest = new AccountVerifyRequest(
            emailAddress: "some_user@dropboxsign.com"
        );

        try
        {
            var response = new AccountApi(config).AccountVerify(
                accountVerifyRequest: accountVerifyRequest
            );

            Console.WriteLine(response);
        }
        catch (ApiException e)
        {
            Console.WriteLine("Exception when calling Account#AccountVerify: " + e.Message);
            Console.WriteLine("Status Code: " + e.ErrorCode);
            Console.WriteLine(e.StackTrace);
        }
    }
}
